deps/graph-parser/src/logseq/graph_parser/util.cljs

```diff
@@ -208,7 +208,7 @@
 
 (defn path->file-ext
   [path-or-file-name]
-  (last (split-last "." path-or-file-name)))
+  (second (re-find #"(?:\.)(\w+)(?:\?.*)?(?:#.*)?$" path-or-file-name)))
```

Comment:
@sallto Could you add a unit test to util_test.cljs so we don't regress on supporting these uris?